pkg/rootless/rootless_linux.go

```diff
@@ -105,7 +105,7 @@ func tryMappingTool(uid bool, pid int, hostID int, mappings []idtools.IDMap) err
 	}
 	path, err := exec.LookPath(tool)
 	if err != nil {
-		return errors.Wrapf(err, "cannot find %s", tool)
+		return errors.Wrapf(err, "cannot find %s required for rootless mode (try running as root?)", tool)
```

Comment:
This is stuttering.

How about just  errors.Wrapf(err, "command required for rootless mode")

Which will result in 

command required for rootless mode:  exec: "newuidmap": executable file not found in $PATH

Reply:
I'd make it clearer as: `errors.Wrapf(err, "command required for rootless mode with multiple IDs")` or similar